docs/articles/documentation/recipes/best-practices/create-faster-tests.md

```diff
@@ -0,0 +1,101 @@
+
+With `Roles`, TestCafe remembers cookies associated with every logged-in account. When you switch Roles, TestCafe erases authentication data and you don't have to log out manually.
+
+If your test covers logins on multiple websites, the active Role collects the authentication data from all these websites. When you switch to that Role later, you are logged in to all these websites.
+
+Use an [Anonymous Role](../../guides/advanced-guides/authentication.md#anonymous-role) to instantly log out of all accounts during a test.
+
+> Roles can access authentication data in cookie and browser storage. If your authentication system stores data elsewhere, you may not be able to use roles.
+
+For more info on Roles, see [User Roles](../../guides/advanced-guides/authentication.md#user-roles)
+
+## Run Tests Concurrently
+
+Tests execute faster if run concurrently. In concurrent mode, TestCafe creates multiple instances of specified browsers and uses that browser pool to run tests.
+
+Use the [--concurency](../../reference/command-line-interface.md#-c-n---concurrency-n) CLI option to launch tests concurrently:
+
+```sh
+testcafe --concurency 3 chrome tests/
+```
+
+For more info on concurrency, read [Run Tests Concurrently](../../guides/basic-guides/run-tests.md#run-tests-concurrently).
+
+## Mock Requests
+
+A tested application may interact with remote resources (for example, an analytics service or a database). Requests to such resources may create delays in test suite execution. To avoid such delays, you can mock requests to these resources.
+
+TestCafe [request mocker](../../reference/test-api/requestmock/README.md) intercepts requests from your app to external resources and responds with data that you specify. A mocked request is resolved almost instantly, which eliminates a possible delay caused by a foreign resource or network lag.
+
+For more information about mocking requests, visit [Mock HTTP Requests](../../guides/advanced-guides/intercept-http-requests.md#mock-http-requests).
+
+## Optimize Your Page Model
+
+Page model is a great way to structure your test suite. However, wrong page model structure can increase testing time in large test suites.
+
+In your page model file, do not export the page model constructor, but create and export a new instance.
+
+```js
+import { Selector } from 'testcafe';
+import { t } from 'testcafe';
+
+class MyPage {
+    //page model contents here
+}
+
+export default new MyPage()
+// creates an instance of MyPage
+// that can be imported in tests
+```
+
+In your test files, avoid the `new` keyword when creating page model instances. Import the page object instead:
+
+```js
+import myPage from 'path/to/page-model.js'
+// a recommended way to import page models
+
+
+//
+const myPage = new MyPage();
+// NOT RECOMMENDED
+// may lead to performance issues in large test suites
```

Comment:
```suggestion
// may lead to performance issues in large fixtures
```

Reply:
A test suite can (and often does) contain multiple fixtures. This approach would only significantly benefit a test suite of [with numerous page models and dozens of tests or more](https://github.com/DevExpress/testcafe/issues/4054#issuecomment-519450486)